### PR TITLE
Remove usage of global imports

### PR DESCRIPTION
**Pull request by darkwebdev**
Now it should be safe to remove global `index.ts` in next major release.

How I've tested:
- removed global index.ts
- yarn build
- yarn test
- yarn storybook
- 👍

### CODE REVIEW

**Review by HenriqueLimas — COMMENTED** [review]
We need to add BREAKING in the title/comment. Should we move to the branch that we are doing the callbacks?

**Comment by darkwebdev** [comment]
@HenriqueLimas no, this PR only removes using of global imports inside our project. It doesn't break anything.

File: src/ebay-menu/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
-import { render, fireEvent } from '@testing-library/react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import { act } from '@testing-library/react-hooks'
```
Comment (HenriqueLimas) [inline src/ebay-menu/__tests__/index.spec.tsx:3]: Act and screen are not being used do we need to import it?

Comment (darkwebdev) [reply src/ebay-menu/__tests__/index.spec.tsx:3]: @HenriqueLimas thanks, removed. I've created an issue regarding that problem recently: #202

**Review by HenriqueLimas — APPROVED** [review]

**Pull request merged**